File: HomeSmart/HomeSmart/Model/Kitchen.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HomeSmart.Model
{
    public class Kitchen
    {
        public string _id { get; set; }
        public string homeId { get; set; }
        public Boolean lamp { get; set; }
        public Boolean waterPump { get; set; }
        public DateTime createdAt { get; set; }
        public DateTime updatedAt { get; set; }
    }
}

File: Server/models/Kitchen.js
const mongoose = require('mongoose')

const KitchenSchema = new mongoose.Schema({
    homeId:{
        type: String,
        required: true
    },
    lamp:{
        type: Boolean,
        default: false,
        // false: Tắt, True: Bật
    },
    waterPump:{
        type: Boolean,
        default: false,
        // false: Tắt, True: Bật
    }
})

module.exports = mongoose.model("Kitchen",KitchenSchema)